### PR TITLE
Seem/config

### PR DESCRIPTION
**Pull request by seem-ish**

### CODE REVIEW

File: krr.bib [hunk 1]
```diff
@@ -8903,6 +8903,14 @@ @article{ferraris11a
   year = {2011}
 }
 
+@incollection{fersch14a,
```
Comment (tortinator) [inline krr.bib:8906]: falsch14a ?

**Review by tortinator — COMMENTED** [review]
Almost perfect :) Thanks for your effort!!!

File: krr.bib (hunk 1 above)
Comment (seem-ish) [reply krr.bib]: Sorry :(

**Review by tortinator — APPROVED** [review]

File: krr.bib (hunk 1 above)
Comment (tortinator) [reply krr.bib]: No worries! :nerd_face:

**Pull request merged**